src/m365/flow/commands/owner/owner-add.spec.ts

```diff
@@ -0,0 +1,193 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { telemetry } from '../../../../telemetry';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+import { aadUser } from '../../../../utils/aadUser';
+import { aadGroup } from '../../../../utils/aadGroup';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Cli } from '../../../../cli/Cli';
+import { formatting } from '../../../../utils/formatting';
+const command: Command = require('./owner-add');
+
+describe(commands.OWNER_ADD, () => {
+  const validEnvironmentName = 'Default-6a2903af-9c03-4c02-a50b-e7419599925b';
+  const validName = '784670e6-199a-4993-ae13-4b6747a0cd5d';
+  const validUserId = 'd2481133-e3ed-4add-836d-6e200969dd03';
+  const validUserName = 'john.doe@contoso.com';
+  const validGroupId = 'c6c4b4e0-cd72-4d64-8ec2-cfbd0388ec16';
+  const validGroupName = 'CLI Group';
+  const validRoleName = 'CanEdit';
+
+  let log: string[];
+  let logger: Logger;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    auth.service.connected = true;
```

Comment:
Could you add the new test stubs?
https://github.com/pnp/cli-microsoft365/pull/4545/files#diff-195628c809e6fb2923588c891e9fe2362a4422873efef5b27532a051cf9d065d